lib/Driver/ToolChains.cpp

```diff
@@ -1047,6 +1047,9 @@ toolchains::GenericUnix::constructInvocation(const LinkJobAction &job,
     break;
   case LinkKind::DynamicLibrary:
     Arguments.push_back("-shared");
+    if (getTriple().getArch() == llvm::Triple::ArchType::arm) {
+      Arguments.push_back("-Wl,-Bsymbolic");
```

Comment:
Do you have a good reference for how to write these tests, I looked, but I couldn't find any good docs.